Content.Shared/Weapons/Ranged/Systems/RechargeBasicEntityAmmoSystem.cs

```diff
@@ -47,8 +47,9 @@ public override void Update(float frameTime)
 
             if (_gun.UpdateBasicEntityAmmoCount(uid, ammo.Count.Value + 1, ammo))
             {
-                if (_netManager.IsClient && _timing.IsFirstTimePredicted)
-                    _audio.Play(recharge.RechargeSound, Filter.Local(), uid, true);
+                // We don't predict this because occasionally on client it may not play.
+                if (_netManager.IsServer)
+                    _audio.PlayPvs(recharge.RechargeSound, uid);
```

Comment:
i *think*  setting the user to null in PlayPredicted would fix this?

Reply:
Client predicts playing it still so it double-plays.